File: src/com/izhan/convertanumbertoalengthstring/java/Convert.java
package com.izhan.convertanumbertoalengthstring;

public class Convert {
    public static Length numberToLength(Number number) {
        return new Length(number.getValue(), "m");
    }

    public static Length numberToLength(Number number, java.lang.String unit) {
        return new Length(number.getValue(), unit);
    }

    public static java.lang.String lengthToString(Length length) {
        return String.fromLength(length);
    }

    public static java.lang.String numberToString(Number number) {
        return String.fromNumber(number);
    }
}

File: src/com/izhan/convertanumbertoalengthstring/android/Convert.kt
package com.izhan.convertanumbertoalengthstring.android

object Convert {
    fun numberToLength(number: com.izhan.convertanumbertoalengthstring.android.Number): com.izhan.convertanumbertoalengthstring.android.Length {
        return com.izhan.convertanumbertoalengthstring.android.Length(number.value, "m")
    }

    fun numberToLength(number: com.izhan.convertanumbertoalengthstring.android.Number, unit: kotlin.String): com.izhan.convertanumbertoalengthstring.android.Length {
        return com.izhan.convertanumbertoalengthstring.android.Length(number.value, unit)
    }
}
